Wikipedia/Code/TalkPageCell.swift

```diff
@@ -109,6 +109,20 @@ final class TalkPageCell: UICollectionViewCell {
 
         stackView.addArrangedSubview(topicView)
     }
+    
+    // MARK: - Public
+    
+    func displayingCommentViewForViewModel(_ commentViewModel: TalkPageCellCommentViewModel) -> TalkPageCellCommentView? {
+        
+        return stackView.arrangedSubviews.first { view in
```

Comment:
I think this entire expression can be simplified and read a little clearer as:
```swift
return stackView.arrangedSubviews
            .compactMap { $0 as? TalkPageCellCommentView }
            .first(where: { $0.viewModel == commentViewModel })
```

Reply:
Good call, this is done!